services/net/contentmigration/ContentMigrationAction.cs

```diff
@@ -200,7 +200,7 @@ public class ContentMigrationAction : IngestAction<ContentMigrationOptions>
         {
             try
             {
-                var baseFilter = contentMigrator.GetBaseFilter();
+                var baseFilter = contentMigrator.GetBaseFilter(manager.Ingest.IngestType!.ContentType);
```

Comment:
Using a `!` is dangerous as it's ignoring the lint warning.  Handle the nullable issue.